config/values.yml

```diff
@@ -25,6 +25,10 @@ images:
   capi: ""
   nginx: ""
   cfroutesync: ""
+  log_cache: "logcache/log-cache"
+  syslog_server: "logcache/syslog-server"
+  log_cache_gateway: "logcache/log-cache-gateway"
+  fluent: "logcache/cf-k8s-logging"
```

Comment:
i dont think we need to expose these configuration to end users. can just set them in config/_ytt_lib/github.com/loggregator/cf-k8s-logging/config/values.yml.

Reply:
👍